### ISSUE TITLE
While extracting cheats, progress bar suddenly jumps to the end... does this mean not all cheats are extracted?

### ISSUE THREAD

**Issue by ms2048** [HamletDuFromage/aio-switch-updater#164]
**Describe the bug**
After selecting the option, the file is (optionally) downloaded and then the 'Extracting...' progress bar appears. It slowly goes up from 0% to 83% and then abruptly jumps to 99% and then the message 'All done!' appears.

If this is just a GUI issue, I have no problem with that. However, I have no way of checking whether it missed some files to be extracted, so I am concerned this might be a symptom of something more severe. Maybe some option for logging would help in determining the severity?

**To Reproduce**
When online, select Download cheats -> download GBAtemp.net cheat archive (ver 11142021) -> Continue

**Screenshots/Crash code/Crash reports**
n/a

**System Firmware Version and CFW Version**
 - System Firmware: 13.1.0
 - CFW Version: Atmosphère 1.2.4
 - AIO Version: 2.16.1

**SD card format**
FAT32

**Additional context**
n/a

**Comment by HamletDuFromage** [comment]
The program fetches all your games, then iterate through the resulting list to check if there are cheat codes for it. If it jumps, typically it means that it checks multiple games faster than the refresh rate of the screen, often time meaning there was no cheat available for those games.

tl;dr: should be working fine

**Comment by HamletDuFromage** [comment]
The code is pretty straight forward, and I think reliable
https://github.com/HamletDuFromage/aio-switch-updater/blob/master/source/extract.cpp#L183

**Comment by HamletDuFromage** [comment]
Wait actually nvm, I might have messed up the display :^)
I'll get on it today

**Comment by ms2048** [comment]
confirmed working, thank you!
